backend/de.metas.business/src/main/java/de/metas/invoice/order/restart/VoidInvoicesRelatedToOrderHandler.java

```diff
@@ -79,7 +79,7 @@ public void handleOrderVoided(@NonNull final VoidOrderAndRelatedDocsRequest requ
 						I_C_Invoice.COLUMNNAME_C_Invoice_ID,
 						invoiceRecord.getDocumentNo(),
 						invoiceDocStatus);
-				throw new AdempiereException(errorMsg);
+				throw new AdempiereException(errorMsg,Services.get(IMsgBL.class).getErrorCode(VoidOrderAndRelatedDocsHandler.Msg_OrderDocumentCancelNotAllowed_4P) );
```

Comment:
loud thinking (u can leave it like this if u want):

IMHO ideally would be to rename createInvalidDocStatusErrorMessage to createInvalidDocStatusException and return AdempiereException